cmake/modules/AddPureSwift.cmake

```diff
@@ -302,6 +302,12 @@ function(add_pure_swift_host_library name)
 
   _set_pure_swift_profile_flags(${name})
 
+  # Enable build IDs
+  if(SWIFT_SDK_${SWIFT_HOST_VARIANT_SDK}_USE_BUILD_ID)
+    target_link_options(${name} PRIVATE
+      "SHELL:-Xlinker --build-id=sha1")
```

Comment:
I would be tempted to suggest to see if `LINKER:SHELL:--build-id=sha1` would work here -- but for the scope of this change it is better to adhere to the existing convention.

Reply:
Yeah, I used the `LINKER:--build-id=sha1` syntax elsewhere, but here I thought it best to match what we already do.